tests/drivers/i2s/i2s_api/src/main.c

```diff
@@ -50,6 +50,7 @@ void test_main(void)
 			ztest_unit_test(test_i2s_state_error_neg));
 	ztest_run_test_suite(i2s_states_test);
 
+#ifndef CONFIG_I2S_MCUX_FLEXCOMM
```

Comment:
The issue with the tests in user mode (an MPU Fault) is not specific to this driver. #35590 should fix it.

Reply:
thanks @anangl